### ISSUE TITLE
Release Twisted Runner 0.2.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1745]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1745|
|Type|task|
|Created|2006-05-24 14:42:48Z|

Yay 0.2.0

<details><summary>Searchable metadata</summary>

```
trac-id__1745 1745
type__task task
reporter__radix radix
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__runner runner
keywords__ 
time__1148481768000000 1148481768000000
changetime__1148515290000000 1148515290000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix removed owner|
|-|-|

Please review [branches/releases/twisted-runner-0.2.x-1745](branches/releases/twisted-runner-0.2.x-1745)

**Comment by twisted-trac** [comment]
|[<img alt="dreid's avatar" src="https://avatars.githubusercontent.com/u/48695?s=50" width="50" height="50">](https://github.com/dreid)|@dreid set owner to @radix|
|-|-|

branch is actually at [branches/releases/twisted-runner-0.2.x-1743](branches/releases/twisted-runner-0.2.x-1743)

Otherwise, Hooray release!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [16932]) Versioning and news for Twisted Runner 0.2.0 release

Author: Release Automation, radix
Reviewer: radix, dreid
Fixes [#1745](https://github.com/twisted/twisted/issues/1745)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
